src/views/About.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styled from 'styled-components';
+import theme from '../theme/styledTheme';
+import luke from '../avatar/luke.jpg';
+import talent from '../avatar/talent.jpeg';
+import diana from '../avatar/diana.jpg';
+import mark from '../avatar/mark.jpg';
+import pavol from '../avatar/pavol.PNG';
+import connor from '../avatar/connor.jpg';
+
+class About extends React.Component {
+  render() {
+    return (  
+      
+      <PageOuter>
+        <PageContainer>
+        <section className="about-container">
+            <div class="card">
+              <img src={luke} alt=""/>
+               <h2>Lukasz Busk</h2>
+               <p>Project Manager</p>
+               <a href="https://github.com/Velsu"><i class="fab fa-github gh"></i></a>
```

Comment:
icons need to be bigger so I'd suggest maybe increasing their font-size to 1.6rem and add a transition on hover like the footer icons

Reply:
Also put something in alt.